main.go

```diff
@@ -76,7 +77,7 @@ func main() {
 				os.Exit(1)
 			}
 
-			break
+			break loop
```

Comment:
Thanks. Break inside select inside for always scares me, I know I'll get it
wrong.

On Fri, Apr 1, 2016 at 10:00 PM, Justin Nuß <notifications@github.com>
wrote:

> In main.go
> <https://github.com/davecheney/gcvis/pull/27#discussion_r58190916>:
>
> > @@ -76,7 +77,7 @@ func main() {
> >  				os.Exit(1)
> >  			}
> >
> > -			break
> > +			break loop
>
> That would also make sense. I will update the code.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/davecheney/gcvis/pull/27/files/ee2a1791eeff5945f48dcdb1d873dec6f1f3b8ee#r58190916>
>